tests/60app-services/03passive.pl

```diff
@@ -22,24 +22,25 @@
             });
          });
 
-      matrix_invite_user_to_room( $creator, $user_id, $room_id )
-         ->SyTest::pass_on_done( "Sent invite" )
```

Comment:
this is pretty redundant, since `matrix_invite_user_to_room` does a log_on_fail when the invite is sent. And since this is the only `pass_on_done`, we may as well convert from a `multi_test` to a regular `test`.